include/GooseEYE/path.hpp

```diff
@@ -13,17 +13,21 @@ namespace GooseEYE {
 
 // -------------------------------------------------------------------------------------------------
 
-xt::xarray<int> path(const std::vector<int> &xa, const std::vector<int> &xb, path_mode mode)
+xt::xtensor<size_t,2> path(
+  const xt::xtensor_fixed<size_t, xt::xshape<3>>& x0,
+  const xt::xtensor_fixed<size_t, xt::xshape<3>>& x1,
+  path_mode mode)
 {
-  GOOSEEYE_ASSERT(xa.size() == xb.size());
-  GOOSEEYE_ASSERT(xa.size() > 0 && xa.size() <= 3);
-
-  int ndim = static_cast<int>(xa.size());
-
-  std::vector<int> ret;
+  xt::xtensor<size_t,2> points;
 
   if (mode == path_mode::Bresenham)
   {
+    bressenham( points, x0, x1 );
+  }
```

Comment:
For me there is no urgency to implement other algorithms, could also be done in a dedicated PR. If left as todo a throw should be included here though